browser-interface/packages/shared/session/index.ts

```diff
@@ -57,7 +58,9 @@ async function getSessionWithSSOIdentity(sessions: StoredSession[]) {
 
   const ssoIdentity = await SingleSignOn.getIdentity(session.identity.address)
 
-  if (!ssoIdentity) {
+  const expiredIdentity = ssoIdentity ? isSessionExpired({ identity: ssoIdentity }) : true
+
+  if (!ssoIdentity || expiredIdentity) {
```

Comment:
```suggestion
  if (!ssoIdentity || isSessionExpired({ identity: ssoIdentity })) {
```
Maybe this is simpler